app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api::V1
+  class LobbiesController < ApplicationController
+
+    def create
+      lobby = Lobby.create
```

Comment:
I often find it confusing to name a local the same name as a method. I often prefer to use `new_lobby` as the variable name in `create` actions if I have a `lobby` method defined elsewhere.